setup.cfg

```diff
@@ -25,10 +25,12 @@ classifiers =
 packages = find:
 python_requires = >=3.11
 install_requires =
+    astroid == 3.3.9    # for `prevent_returning_fastapi_httpexceptions.py`
```

Comment:
Can't this consume requirements.txt?

Reply:
Solved by moving to `uv` and `tox-uv` which can run tests in an env w/ `uv.lock` dependencies installed.